### ISSUE TITLE
Custom/Agnostic counter

### ISSUE THREAD

**Issue by kailinprime** [Orcpub/orcpub#441]
**Motivation / Use Case**

There are times when a simple counter with a free form description would be handy. For instance, a character picks up the "Widget of widgetiness" and there is some description along with some number of daily uses. To make that, I shouldn't have to go in a create a brand new item just to include it for some one off, and when I do I am not certain if there is even a way to pull in the special "X times per day" counter when it is being added to a character.

I would love it if I could add this to a character on the fly and that counter could be part of the class features when I am playing and it would be even better if I could assign a hook for the reset trigger, (turn/short/long/day/etc)
 
Finally, it would be nice to maintain these are counters that can be saved to the custom content and added from a library, or added on the fly in the character editor.

**Expected Behavior**

From the equipment section in the character editor, add a custom counter. The counter should have a name, a description, and option to include a number of check boxes, and an optional reset trigger option). This will allow a character to add those "3 times a day you can see through green jello but not red jello" special items and have the check boxes reset when a duration trigger longer that 'short rest' was hit.

I think the duration triggers are Turn/Short/Long/Day/Ever(none)

The check boxes should be something that displays in a logical fashion. Like when a wand has 50 charge, it might be in rows of 10.

For this, there should also be an option to choose from a list of possible options: Like when a lair effect is chosen from a few options, but they are mutually exclusive. A or B or C. And those options should be allowed to cost different amounts and pull from a total. In that case, there might be a radio button and a total number points that are subtracted from when viewing the character / monster, but the points would reset on the interval.

**Other Information**

For that matter, when an item has a certain number of uses per day, any item that has a limited number of uses should have a check box and description for those special charged features.

If you wanted to get fancy, you could work that into the class/race builder too, using the same type of paradigm.

The bottom line is that you could use a similar type of approach for any feature of a class, race, item, or one-off random counter that a character might need and they could all present the options for checking off the points. 

Examples: 

A Ki counter for Monks, added to the class builder
A lair effect for a dragon with options of which one is being used, with different options taking variable points worth.
A one-off special ability granted by a shrine for the next week with a twice per day power
The custom drop down that is missing.
